src/language/ast.ts

```diff
@@ -288,7 +292,13 @@ export const QueryDocumentKeys: {
   UnionTypeExtension: ['name', 'directives', 'types'],
   EnumTypeExtension: ['name', 'directives', 'values'],
   InputObjectTypeExtension: ['name', 'directives', 'fields'],
-  SchemaCoordinate: ['name', 'memberName', 'argumentName'],
+
+  // Schema Coordinates
+  TypeCoordinate: ['name'],
+  MemberCoordinate: ['name', 'memberName'],
```

Comment:
Can we call this `EnumMemberCoordinate`, `MemberCoordinate` feels like it could be a field or an enum member

Reply:
I was wondering the same. Maybe we need `FieldCoordinate`, `InputFieldCoordinate` and `EnumMemberCoordinate`?

Reply:
Leaving as is for now until there's consensus on this.

I'm leaning towards keeping it as is for the following 2 reasons:

(1) This distinction is only able to be understood in the context of a schema, which we don't yet have at the time of parsing the schema coordinate. (Even though the spec says coordinates are only valid in the context of a schema, which I think is a fine as a semantic, we may still want to be able to construct an "invalid" coordinate in code as has been pointed out previously.)
(2) It would lead to a messier implementation, since parsing the schema coordinate now needs to take in the schema object too.

Reply:
Good points 👍